pages/documenti.tsx

```diff
@@ -0,0 +1,14 @@
+// pages/documenti.js
+import Head from 'next/head';
+
+const Documenti = () => {
```

Comment:
When the user navigate to /documenti, he sees a blank page for a moment and, after a refresh, the correct page. To avoid this it's better to use the nextsjs router

```
const Documenti = () => {
    const router = useRouter();

    useEffect(() => {
        router.replace('pubbliche-amministrazioni/documenti');
    }, [])

    return (
        <>
            <Head>
                <meta key="robots" name="robots" content="noindex, nofollow" />
            </Head>
        </>
    );
};
```